src/redux/store.ts

```diff
@@ -0,0 +1,42 @@
+import { configureStore, combineReducers } from '@reduxjs/toolkit'
+import {
+  persistStore,
+  persistReducer,
+  FLUSH,
+  REHYDRATE,
+  PAUSE,
+  PERSIST,
+  PURGE,
+  REGISTER
+} from 'redux-persist'
+import AsyncStorage from '@react-native-async-storage/async-storage'
+import currentConnectorSlice from '/redux/ConnectorState/CurrentConnectorSlice'
+import connectorLogsSlice from '/redux/ConnectorState/ConnectorLogsSlice'
+
+const persistConfig = {
+  key: 'root',
+  storage: AsyncStorage
+}
+
+const rootReducter = combineReducers({
+  currentConnector: currentConnectorSlice,
+  connectorLogs: connectorLogsSlice
+})
+
+const persistedReducer = persistReducer(persistConfig, rootReducter)
+
+export const store = configureStore({
+  reducer: persistedReducer,
+  middleware: getDefaultMiddleware =>
+    getDefaultMiddleware({
+      serializableCheck: {
+        ignoredActions: [FLUSH, REHYDRATE, PAUSE, PERSIST, PURGE, REGISTER]
```

Comment:
I still don't understand this need. Is it because of TS?

Reply:
OK next time I'll read the commit message https://github.com/cozy/cozy-flagship-app/pull/516/commits/d11100b0df56e6108d920ab7d49ec39741514ee2 

So now my question is: why do we use redux-tookit?

Reply:
Woups I missed this notification.

I used it because it is the official recommendation: https://redux.js.org/introduction/getting-started